lib/src/date-input/DateInput.tsx

```diff
@@ -212,4 +228,8 @@ const StyledPopoverContent = styled(Popover.Content)`
   }
 `;
 
+const DateInputContainer = styled.div<{ margin: DateInputPropsType["margin"]; size: DateInputPropsType["size"] }>`
+  width: ${(props) => calculateWidth(props.margin, props.size)};
```

Comment:
There is no need to use calculate width or the size.
` ${(props) => props.size == "fillParent" && "width: 100%;"}`
If 1005 is applied to the width when the size is fillParent fixes the weird behavior.